### ISSUE TITLE
Luxon Edm.Date not working

### ISSUE THREAD

**Issue by brandonryan** [odata2ts/odata2ts#278]
When I use the Luxon converter, the resulting data type is "string" for a type "Edm.Date" in the model, and at runtime, it is also a string. The converter seems to not be applied for that specific case.

I have also tried to explicitly "use" each converter in the luxon package.
If you need a reproduction, let me know.

Config:
```ts
import { ConfigFileOptions, EmitModes } from "@odata2ts/odata2ts";

const config: ConfigFileOptions = {
    converters: ["@odata2ts/converter-luxon"],
    services: {
        codex: {
            serviceName: "Codex",
            allowRenaming: true,
            sourceUrl: "/odata",
            source: "src/_gen/codex-odata.csdl.xml",
            output: "src/_gen/codex-odata",
            emitMode: EmitModes.ts,
            skipComments: true,
        }
    }
};

export default config;
```

Partial OData CSDL
```xml
<?xml version="1.0" encoding="utf-8"?>
<edmx:Edmx Version="4.0" xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx">
  <edmx:DataServices>
    <Schema Namespace="Torch.Codex.Server.Api.Training" xmlns="http://docs.oasis-open.org/odata/ns/edm">
      <ComplexType Name="TrainingStatusDto">
        <Property Name="RequirementId" Type="Edm.Int32" Nullable="false" />
        <!-- Here is the Edm.Date -->
        <Property Name="DueDate" Type="Edm.Date" Nullable="false" />
        <Property Name="ContractId" Type="Edm.Int32" Nullable="false" />
        <Property Name="ContractName" Type="Edm.String" Nullable="false" />
        <Property Name="Trainee" Type="Torch.Codex.Server.Api.Training.TrainingStatusPerson" Nullable="false" />
        <Property Name="CourseId" Type="Edm.Int32" Nullable="false" />
        <Property Name="CourseName" Type="Edm.String" Nullable="false" />
        <Property Name="Status" Type="Torch.Codex.Server.Api.Training.TrainingStatusCompletion" Nullable="false" />
        <Property Name="CompletionId" Type="Edm.Int32" />
        <Property Name="HasAttachment" Type="Edm.Boolean" Nullable="false" />
        <Property Name="CompletionDate" Type="Edm.DateTimeOffset" />
        <Property Name="Verifier" Type="Torch.Codex.Server.Api.Training.TrainingStatusPerson" />
        <Property Name="VerificationDate" Type="Edm.DateTimeOffset" />
      </ComplexType>
    </Schema>
  </edmx:DataServices>
</edmx:Edmx>
```

Resulting Model:
```ts
import { DateTime } from "luxon";

export interface TrainingStatusDto {
    requirementId: number;
    // Expecting this to be DateTime as well
    dueDate: string;
    contractId: number;
    contractName: string;
    trainee: TrainingStatusPerson;
    courseId: number;
    courseName: string;
    status: TrainingStatusCompletion;
    completionId: number | null;
    hasAttachment: boolean;
    completionDate: DateTime | null;
    verifier: TrainingStatusPerson | null;
    verificationDate: DateTime | null;
}
```

**Comment by texttechne** [comment]
Hi @brandonryan,
this has been fixed with [@odata2ts/odata2ts@0.36.3](https://github.com/odata2ts/odata2ts/releases/tag/%40odata2ts%2Fodata2ts%400.36.3).

Please test and close this issue if it works for you.

**Comment by brandonryan** [comment]
This is fixed! Thank you!
